src/components/Calculator.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './Calculator.css';
+
```

Comment:
- [ ] The Calculator component should import the calculate.js file from the logic directory, please do so to comply with the project requirements.